setup.py

```diff
@@ -103,8 +103,9 @@ def main():
             "Programming Language :: Python :: 3.11",
             "Programming Language :: Python :: 3.12",
             "Programming Language :: Python :: 3.13",
+            "Programming Language :: Python :: 3.14",
         ],
-        python_requires=">=3.9,<3.14",
+        python_requires=">=3.9,<3.15",
```

Comment:
we probably don't wan to make this change until python 3.14 is green, right?

Reply:
Correct. None of the workflow jobs, except for pip-install, actually depend on this change.

Reply:
yes, but we don't want this particular change checked in because we may want to cut a release in the next week or so and the python_requires would incorrectly say that parsons supports 3.14.

Reply:
Exactly.

Reply:
Not actually an issue yet since the workflow can't even get past installing dependencies 🤷🏻‍♀️